ui/desktop/electron-app/src/index.js

```diff
@@ -173,8 +174,7 @@ app.on('ready', async () => {
       : path.normalize(`${emberAppDir}${absolutePath}`);
 
     if (isDev) console.log('[serving]', request.url);
-
-    callback({ path: normalizedPath });
+    return net.fetch(`file:///${normalizedPath}`);
```

Comment:
I haven't tested this but should it be 
```suggestion
    return net.fetch(`file://${normalizedPath}`);
```

Doesn't the `normalizedPath` include a leading slash?

Reply:
normalizedPath has a leading slash, interestingly, it didn't throw any error with three vs four slashes(incl the normalizedPath), could it be because the file protocol ignores the extra slash?

Reply:
Honestly it's a little confusing, there's a literal [section](https://en.wikipedia.org/wiki/File_URI_scheme#How_many_slashes?) on wikipedia on how many slashes 🤷